npm-init.js

```diff
@@ -1,4 +1,28 @@
 const fs = require('fs');
+const execSync = require('child_process').execSync;
+
+const gitUser = execSync('git config user.name').toString().replace('\n','');
```

Comment:
I like this.
my machine username is different from github username so this solves the problem fine.